views/component/nav/template.njk

```diff
@@ -12,7 +12,7 @@
                 <a class="nav-item govuk-link {% if originalUrl === '/content-management' %}active{% endif %}" href="/content-management">Admin</a>
             {% endif %}
             {% if identity.isReporter() %}
-                <a class="nav-item govuk-link {% if originalUrl === '/reporting' %}active{% endif %}" href="/reporting">Reporting</a>
+                <a class="nav-item govuk-link {% if originalUrl and originalUrl | startsWith('/reporting') %}active{% endif %}" href="/reporting">Reporting</a>
```

Comment:
is it the correct syntax for the condition?

Reply:
Yes, the startsWith function is a Nunjucks filter and is used in templates with the pipe syntax